geoh5py/groups/property_group.py

```diff
@@ -61,7 +61,7 @@ def __init__(
 
         self._parent: ObjectBase = parent
         self._properties: list[uuid.UUID] | None = None
-        self._property_group_type = "Multi-element"
+        self._property_group_type = kwargs.get("property_group_type", "Multi-element")
```

Comment:
It would be better to add it to the list of optional arguments, so we can document it.